src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let man = people.filter(p => p.sex === 'm');
+
+  if (century) {
+    man = people.filter(p => p.sex === 'm'
+    && century === Math.ceil(p.died / 100));
+  }
+
+  const life = man.map(male => male.died - male.born);
+  const ageSum = life.reduce((sum, age) => sum + age);
+  const averageAge = ageSum / life.length;
+
+  return averageAge;
```

Comment:
you're using the same logic over few times
let's make it a helper function and use it where needed.